src/ApiManagement/ApiManagement/Commands/NewAzureApiManagement.cs

```diff
@@ -61,7 +61,13 @@ public class NewAzureApiManagement : AzureApiManagementCmdletBase
         [Parameter(
             ValueFromPipelineByPropertyName = true,
             Mandatory = false,
-            HelpMessage = "The tier of the Azure API Management service. Valid values are Developer, Consumption, Basic, Standard and Premium . Default value is Developer")]
+            HelpMessage = "The tier of the Azure API Management service. Valid values are : " +
```

Comment:
Import-Module platyPS
Import-Module "azure-powershell\artifacts\Debug\Az.ApiManagement\Az.ApiManagement.psd1"
New-MarkdownHelp -Module Az.ApiManagement -OutputFolder .\docs

Generated markdown file is different with the one you manually edited below.